python-package/pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [build-system]
 requires = [
-    "hatchling>=1.12.1"
+    "hatchling>=1.12.1",
+    "pip>=22.0"
```

Comment:
This installs a copy of the latest `pip` inside an isolated build environment; it does not replace the `pip` in the host system.

Reply:
Now that we are using `packaging` instead, think we can mark this thread resolved